mxcube3/routes/limsutils.py

```diff
@@ -230,7 +226,7 @@ def select_proposal(proposal):
     if proposal_info:
         mxcube.session.proposal_code = proposal_info.get('Proposal').get('code', '')
         mxcube.session.proposal_number = proposal_info.get('Proposal').get('number', '')
-        mxcube.session.session_id = proposal_info.get('Session')[0].get('session').get('sessionId')
+        mxcube.session.session_id = proposal_info.get('Session')[0].get('sessionId')
```

Comment:
we have this working. So there is (again) a mismatch in ispyb data between us. arggghhh!!!

Reply:
That seems really strange to me, because that data should be returned canonically by the IPSyB web-services. Ill check with Alejandro (our ISPyB developer) to see if there is the possibility of differences in the internal database structure that might cause this kind of problem.